Docs/TODO.md

```diff
@@ -4,37 +4,38 @@
 
 ## Gameplay Tweaks
 
-- Make Fleshpound Avoid Area's only spawn when enraged and make Avoid Area extend a couple of meters infront of Fleshpounds to make zeds "Clear the way"
+- [ ] Make Fleshpound Avoid Area's only spawn when enraged and make Avoid Area extend a couple of meters infront of Fleshpounds to make zeds "Clear the way"
   - Currently, Fleshpound Avoid Area has been temporarily removed until this is done.
-- Fleshpound/Ranged Pound head hitboxes too generous
-- Bloat acid stays for far too long, make it go away earlier.
+- [ ] Fleshpound / Ranged Pound head hitboxes too generous
+- [ ] Bloat acid stays for far too long, make it go away earlier.
```

Comment:
This file has been updated, but I'll hold off on pushing any further commits on this until I can pull the directory changes.